demos/browser/server.js

```diff
@@ -206,7 +217,59 @@ function serve(host = '127.0.0.1:8080') {
           console.warn("Cloud media capture not available")
           respond(response, 500, 'application/json', JSON.stringify({}))
         }
-      } else if (request.method === 'POST' && requestUrl.pathname === '/deleteAttendee') {
+      }
+      else if (request.method === 'POST' && requestUrl.pathname === '/startLiveConnector') {
```

Comment:
Suggest doing `try..catch` here in order to catch any errors in logs.

Reply:
will do